src/components/Footer/constants.ts

```diff
@@ -21,17 +22,21 @@ type SocialLink = {
 
 export const FOOTER_LINKS: { home: Link[]; app: Link[] } = {
   home: [
-    { text: t`Terms and Conditions`, link: "/terms-and-conditions" },
-    { text: t`Referral Terms`, link: "/referral-terms" },
-    { text: t`Media Kit`, link: "https://gmxio.gitbook.io/gmx/media-kit", external: true },
+    { text: defineMessage({ message: "Terms and Conditions" }), link: "/terms-and-conditions" },
+    { text: defineMessage({ message: "Referral Terms" }), link: "/referral-terms" },
+    { text: defineMessage({ message: "Media Kit" }), link: "https://gmxio.gitbook.io/gmx/media-kit", external: true },
     // { text: "Jobs", link: "/jobs", isAppLink: true },
   ],
   app: [
-    { text: t`Media Kit`, link: "https://gmxio.gitbook.io/gmx/media-kit", external: true },
+    { text: defineMessage({ message: "Media Kit" }), link: "https://gmxio.gitbook.io/gmx/media-kit", external: true },
```

Comment:
Sorry, why we need to change `t'...'` to defineMessage here? What's the difference?

Reply:
`defineMessage` is used for lazy translation which means the messages doesn't have to be defined at the same code location where they are displayed.

https://lingui.js.org/tutorials/react-patterns.html#lazy-translations